backend/api/models.py

```diff
@@ -111,6 +111,9 @@ def __str__(self):
 class ServerUser(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     host = models.URLField()
+    prefix = models.CharField(max_length=20, blank=True)
```

Comment:
what is this prefix referring to?

Reply:
for example:
our team is host/api/friendrequest
bennet team is host/friendrequest